week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/type.ts

```diff
@@ -0,0 +1,27 @@
+
+export type ReturnType = HTMLDivElement | HTMLElement | HTMLParagraphElement
+
+export  type CartItem = {
+    name: string
+    amount: number
+    unit: string,
+    price: {
+        value: number
+        currency: string
+    }
+}
+
+export type CartItemSum = {
+    value: number
+    currency:  "PLN"
+}
+
+export type CartPanel = {
+    heading?: string
+    items: CartItem[]
+}
+
+export type HeroType = {
+    title: string,
+    subTitle: string
+}
```

Comment:
To że masz te `type` jest super. Natomiast nie pakowałbym wszystkich typów w jedno miejsce, do `types.ts` z powodzeniem mogą one Ci się przydać @DragoBJJ - bezpośrednio w plikach których dotyczą i nie być w ogóle `export`'owane.

Zauważ że część z nich to po prostu tzw. `PropTypes` - określają kształt danych jedynie dla komponentu.
I jedynie w środku niego ich używasz. Nigdzie indziej  dodatkowo ich nie stosujesz.

np. `HeroType` mogołoby być z powodzeniem wrzucone do `hero.component.ts`